service/src/main/java/uk/gov/hmcts/reform/fpl/service/validators/CaseSubmissionChecker.java

```diff
@@ -50,10 +45,7 @@
 
     @Override
     public List<String> validate(CaseData caseData) {
-        if (featureToggleService.isApplicationDocumentsEventEnabled()) {
-            return super.validate(caseData, REQUIRED_EVENTS_AFTER_APPLICATION_DOCUMENTS_ENABLED);
-        }
-        return super.validate(caseData, REQUIRED_EVENTS);
+        return super.validate(caseData, REQUIRED_EVENTS_AFTER_APPLICATION_DOCUMENTS_ENABLED);
```

Comment:
actually we can change this back to 'REQUIRED_EVENTS' and remove 'REQUIRED_EVENTS_AFTER_APPLICATION_DOCUMENTS_ENABLED'.. in required events just update the old DOCUMENTS enum to now be APPLICATION DOCUMENTS